Client/Frontend/Browser/Tab Tray/TabTrayController.swift

```diff
@@ -515,6 +529,55 @@ class TabTrayController: LoadingViewController {
     tabManager.removeTabsWithUndoToast(tabManager.tabsForCurrentMode)
     applySnapshot()
   }
+  
+  private func presentSyncSettings(status: TabSyncContainerView.SyncActionType) {
+    switch status {
+      case .noSyncChain:
+        openInsideSettingsNavigation(
+          with: SyncWelcomeViewController(
+            syncAPI: braveCore.syncAPI,
+            syncProfileServices: braveCore.syncProfileService,
+            tabManager: tabManager))
+      case .openTabsDisabled:
+        if !DeviceInfo.hasConnectivity() {
+          present(SyncAlerts.noConnection, animated: true)
+          return
+        }
+      
+        openInsideSettingsNavigation(with:
```

Comment:
For this case, would we be able to enable tab sync without opening sync settings and having the user to do this manually?